main.tf

```diff
@@ -30,43 +30,41 @@ module "grafana" {
 
   count = var.grafana_configs.enabled ? 1 : 0
 
+  chart_version          = var.grafana_configs.chart_version
   grafana_admin_password = var.grafana_admin_password
   configs                = var.grafana_configs
-  aws_region             = var.aws_region
-  namespace              = var.namespace
+  datasources            = var.grafana_configs.datasources
```

Comment:
we maybe can automatically create loki/tempo/prometheus data sources here if this tools are enabled, with also possibility to have option to pass additional data sources, so that this line will be something like:
```tf
....
datasources = merge(var.grafana_configs.datasources,
var.prometheus_configs.enabled ? {type: "prometheus"} : {},
var.loki_configs.enabled ? {type: "liki"} : {},
...

)
....
```